omise/__init__.py

```diff
@@ -1,37 +1,19 @@
 import copy
 import logging
```

Comment:
This is now unused, can you remove it alongside with the `logger` stuff in here?

Reply:
@sirn sorry, forgot to remove 🙇